app/swagger-api/Main.hs

```diff
@@ -0,0 +1,21 @@
+-- SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
+--
+-- SPDX-License-Identifier: MPL-2.0
+
+module Main where
+
+import Control.Lens
+import Data.Aeson.Encode.Pretty (encodePretty)
+import Data.ByteString.Lazy.Char8 qualified as BL
+import Data.Data (Proxy(Proxy))
+import Data.OpenApi
+import Servant.OpenApi (toOpenApi)
+import Web.API (API)
+
+main :: IO ()
+main = do
+  let openApi = toOpenApi (Proxy :: Proxy API)
+        & info . title .~ "Coffer Web API"
+        & info . version .~ "1.0"
+        & servers .~ ["localhost:8081"]
+  BL.writeFile "swagger.json" (encodePretty openApi)
```

Comment:
If this works, we should add a pipeline step to make sure the committed swagger.json is up to date.
I'll write an issue for this later, I'm just leaving this comment here as a reminder for myself.

Reply:
Done

Reply:
Hmm, the auto-generated json doesn't contain the license.

I think it would be convenient to automatically add the license. Do I need to just edit the `app/swagger-api/Main.hs` so that the license is concatenated with the generated `json` before writing to the file?

Reply:
Yup, I've added this file to `.reuse/dep5`